clusters/hive/pools/msp/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+  - scoheb
+  - apodhrad
+  - kpavic
+  - jupierce
+  - skuznets
```

Comment:
These should be GH logins

Reply:
thanks, using the rover I found correct GH logins

Reply:
names were updated